### PR TITLE
setup `tox` to run test in multiple `python` versions

### PR DESCRIPTION
**Pull request by hotay**
This PR address https://github.com/EastAgile/robber.py/issues/10

### CODE REVIEW

**Comment by hotay** [comment]
@tklarryonline @steadykhoi @nhanbt @hieueastagile please help me review this one.

File: .python-version [hunk 1]
```diff
@@ -0,0 +1,4 @@
+2.6.9
```
Comment (hieueastagile) [inline .python-version]: We don't need this, I think, but an instruction how we use `pyenv` and `pyenv local` instead.

File: README.md [hunk 2]
```diff
@@ -343,6 +343,23 @@ $ pip install robber
...
+```
+- Install `tox`
+```
+pip install tox
+```
+- Run test:
+```
+tox
+```
+
```
Comment (hieueastagile) [inline README.md]: Need to add the instruction for creating the corresponding versions of python with `pyenv` and use it with `pyenv local`.

**Comment by tklarryonline** [comment]
Linking this to Issue #10

File: .python-version (hunk 1 above)
Comment (hotay) [reply .python-version]: :+1:

**Pull request merged**